autoscaler/example-rc.yaml

```diff
@@ -0,0 +1,36 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: pod-autoscaler-nanny-params
+  namespace: default
+data:
+  params: |-
+    { "cores_to_replicas_map": { "1":1, "2":2, "3":3, "512":5, "1024":7, "2048":10, "4096":15, "8192":20, "12288":30, "16384":40, "20480":50, "24576":60, "28672":70, "32768":80, "65535":100} }
```

Comment:
Suggest making the key name more explicit.  As of a couple days ago, they key can be much more freeform, so perhaps "simple-scalar.scale-map" so we can store it in the same configmap as DNS config itself?